src/main/java/org/broadinstitute/consent/http/ConsentApplication.java

```diff
@@ -249,17 +249,21 @@ public void run(ConsentConfiguration config, Environment env) {
 
     // Authentication filters
     final UserRoleDAO userRoleDAO = injector.getProvider(UserRoleDAO.class).get();
-    AuthFilter defaultAuthFilter = new DefaultAuthFilter.Builder<AuthUser>()
-        .setAuthenticator(new DefaultAuthenticator())
-        .setRealm(" ")
-        .buildAuthFilter();
-    List<AuthFilter> filters = List.of(
-        defaultAuthFilter,
-        new OAuthCustomAuthFilter(authenticator, userRoleDAO));
+    // Requests annotated with @Auth AuthUser will be authenticated with this filter
+    final AuthFilter<String, AuthUser> primaryAuthFilter = new OAuthCustomAuthFilter<>(authenticator, userRoleDAO);
+    // Requests annotated with @Auth DUOSAuthUser will be authenticated with this filter and are guaranteed to have a populated User object
+    final AuthFilter<String, DUOSAuthUser> duosAuthUserFilter = new OAuthCustomAuthFilter<>(authenticator, userRoleDAO);
+    final PolymorphicAuthDynamicFeature<AuthUser> feature = new PolymorphicAuthDynamicFeature<>(
+        ImmutableMap.of(
```

Comment:
Instead of google APIs, I think we should use JDK APIs where possible. In this case the code can use `Map.of()` and `Set.of()` instead.
```suggestion
        Map.of(
```